lib/dnsimple/struct/certificate.rb

```diff
@@ -23,6 +26,9 @@ class Certificate < Base
       # @return [String] The Certificate Authority (CA) that issued the certificate.
       attr_accessor :authority_identifier
 
+      # @return [TrueClass,FalseClass] The certificate auto renew flag.
+      attr_accessor :auto_renew
+
```

Comment:
@jodosha while working on this I realized that the payload (according to `purchaseLetsencryptCertificate/success.http`) contains the `contact_id` but it's not in the `Certificate` struct. Is this on purpose?

Reply:
@jacegu Please hold on with the Elixir client, we're discussing internally about this feature. Thanks 😄

Reply:
Sure 👍  Please, ping me when the discussion is over so I can move forward with the work I've started. Thanks! 😃